docs/en/sql-reference/sql-statements/Data Definition/CREATE TABLE.md

```diff
@@ -405,68 +409,89 @@ Syntax:
     "storage_cooldown_time" = "2015-06-04 00:00:00"
     );
     ```
-
-3. Create an olap table, with range partitioned, distributed by hash. Records with the same key exist at the same time, set the initial storage medium and cooling time, use default column storage.
-
-1) LESS THAN
-
+   
+3. Create an olap table, distributed by hash, with aggregation type. Also set storage medium and cooldown time.
+   Setting up remote storage resource and cold data storage media.
     ```
-    CREATE TABLE example_db.table_range
+    CREATE TABLE example_db.table_hash
     (
-    k1 DATE,
-    k2 INT,
-    k3 SMALLINT,
-    v1 VARCHAR(2048),
-    v2 DATETIME DEFAULT "2014-02-04 15:36:00"
+    k1 BIGINT,
+    k2 LARGEINT,
+    v1 VARCHAR(2048) REPLACE,
+    v2 SMALLINT SUM DEFAULT "10"
     )
     ENGINE=olap
-    DUPLICATE KEY(k1, k2, k3)
-    PARTITION BY RANGE (k1)
-    (
-    PARTITION p1 VALUES LESS THAN ("2014-01-01"),
-    PARTITION p2 VALUES LESS THAN ("2014-06-01"),
-    PARTITION p3 VALUES LESS THAN ("2014-12-01")
-    )
-    DISTRIBUTED BY HASH(k2) BUCKETS 32
+    AGGREGATE KEY(k1, k2)
+    DISTRIBUTED BY HASH (k1, k2) BUCKETS 32
     PROPERTIES(
-    "storage_medium" = "SSD", "storage_cooldown_time" = "2015-06-04 00:00:00"
+    "storage_medium" = "SSD",
+    "storage_cold_medium" = "S3",
+    "remote_storage_resource" = "remote_s3",
```

Comment:
cold_storage_resource = "xxx,xxxx,xxxx" it should support a array, because S3 bucket's bandwidth is too small, maybe we need multiple bucket to extend the bandwidth.

Reply:
Use the same resource when creating a table to facilitate data management.
Since the `remote_storage_resource` is partition level, user can change the resource when creating a new partition or modify by `ALTER TABEL` clause. In this way, a table can use multiple buckets.

Reply:
If user could specify multi resources then he do not care the bandwidth in the future. If there is only one resource he  need to monitor and optimize the resource usage by using alter table clause. It is too hard for the user.